build.rs

```diff
@@ -18,7 +19,22 @@ fn main() {
                         .join(wat.file_name().unwrap())
                         .with_extension("wasm");
                     let binary = wat::parse_file(&wat).expect("Can't parse wat file");
-                    std::fs::write(wasm, &binary).expect("Can't write wasm file");
+                    std::fs::write(&wasm, &binary).expect("Can't write wasm file");
+                    // If the "enarx" command is installed, create a
+                    // bundled Wasm file for testing.
+                    let status = Command::new("enarx")
+                        .arg("wasm")
+                        .arg("bundle")
+                        .arg("fixtures/bundle")
+                        .arg(&wasm)
+                        .arg(&wasm.with_extension("bundled.wasm"))
+                        .status();
+
+                    if let Ok(status) = status {
+                        assert!(status.success());
```

Comment:
If the command fails, would we get its output when the assert is triggered? If not, that might be helpful for debugging purposes.

Reply:
Indeed, added a bit more diagnostics.